src/WebManager/app/src/services/docker.ts

```diff
@@ -56,6 +56,46 @@ export class Docker {
             });
     }
 
+
+    getManifestHeaders(repo: string, tag: string, cancel: CancelToken = null):
+        Promise<{ manifest: string }> {
+        let cred: RegistryCredentials = this.credService.getRegistryCredentials(this.registryName);
+        if (!cred) {
+            return Promise.resolve({ manifest: null });
+        }
+
+        let config: AxiosRequestConfig = {
+            cancelToken: cancel,
+            baseURL: this.registryEndpoint,
+            params: {},
+            headers: {
+                "Registry": this.registryName,
+                "Access-Control-Expose-Headers": "X-Ms-Request-Id",
+                "Accept": "application/vnd.docker.distribution.manifest.v2+json; 0.6, " +
+                "application/vnd.docker.distribution.manifest.v1+json; 0.5",
+                "Authorization": "Basic " + cred.basicAuth
+            }
+        };
+
+        return axios.get(`/v2/${repo}/manifests/${tag}`, config)
+            .then((r: AxiosResponse) => {
+
```

Comment:
NIT: Format and clean extra lines. Maybe running some kind of linter might be helpful for all formatting issues.